scalanativelib/api/src/GetFrameworkResult.java

```diff
@@ -0,0 +1,14 @@
+package mill.scalanativelib.api;
```

Comment:
This source files should be located in a directory structure reflecting the package name. The Java Standard isn't as flexible as the Scala one. Beside the fact that it just works with zinc, it won't for pure Java compiler.

Same, for all other Java files.

Reply:
@lefou You're right! Moved them ✅